app/Models/Leaderboard.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+
+namespace Proto\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Leaderboard extends Model
+{
+    protected $table = 'leaderboards';
```

Comment:
This isn't needed as the model name is the singular of the table name

Reply:
I prefer to keep it explisit, just like you can see in the code_quality branch.